src/Service/Modules/ParcelsModule.cs

```diff
@@ -44,6 +45,18 @@ private object GetParcels()
                 .WithView("Index");
         }
 
+        private object GetParcelsByNumber()
```

Comment:
Could you combine these two GET endpoints and decide what to do based on the values of the binded resource? Seems odd to have 'parcels' and 'parcelsByNumber' as two seperate js itemTypes when they are the same thing, and the only reason I can see for you seperating them is this different GET endpoint

Reply:
I think I did it because of the action types on the front end - the normal parcels was updateApiActions and parclesbyNumber uses fetchApiActions. The fetchByQueryString in updateApiActions was ideal for the mad search string of multiple things I did in the actual parcels search, but it was going to be a pain to use for this where I just want to be able to call the standard search method without setting up a string like in fetchApiActions. I maybe could use the same resource and method on the backend here, and just check whether it has just the searchTerm or all the other search terms, but then would the front end be smart enough for two sets of actions & selectors for the same url/type? I could try it